lib/crypto.js

```diff
@@ -353,95 +353,87 @@ util.inherits(Verify, stream.Writable);
 Verify.prototype._write = Sign.prototype._write;
 Verify.prototype.update = Sign.prototype.update;
 
-Verify.prototype.verify = function verify(options, signature, sigEncoding) {
-  var key = options.key || options;
-  sigEncoding = sigEncoding || exports.DEFAULT_ENCODING;
+Verify.prototype.verify = function verify(object, signature, signatureFormat) {
+  var key = object.key || object;
+  signatureFormat = signatureFormat || exports.DEFAULT_ENCODING;
 
   // Options specific to RSA
   var rsaPadding = constants.RSA_PKCS1_PADDING;
-  if (options.hasOwnProperty('padding')) {
-    if (options.padding === options.padding >> 0) {
-      rsaPadding = options.padding;
+  if (object.hasOwnProperty('padding')) {
+    if (object.padding === object.padding >> 0) {
+      rsaPadding = object.padding;
     } else {
       throw new TypeError('padding must be an integer');
     }
   }
 
   var pssSaltLength = constants.RSA_PSS_SALTLEN_AUTO;
-  if (options.hasOwnProperty('saltLength')) {
-    if (options.saltLength === options.saltLength >> 0) {
-      pssSaltLength = options.saltLength;
+  if (object.hasOwnProperty('saltLength')) {
+    if (object.saltLength === object.saltLength >> 0) {
+      pssSaltLength = object.saltLength;
     } else {
       throw new TypeError('saltLength must be an integer');
     }
   }
 
-  return this._handle.verify(toBuf(key), toBuf(signature, sigEncoding),
+  return this._handle.verify(toBuf(key), toBuf(signature, signatureFormat),
                              rsaPadding, pssSaltLength);
 };
 
-function rsaPublic(method, defaultPadding) {
-  return function(options, buffer) {
-    var key = options.key || options;
-    var padding = options.padding || defaultPadding;
-    var passphrase = options.passphrase || null;
+function rsaBinder(method, defaultPadding) {
```

Comment:
Um … “binder”?

Elsewhere in the code we name these things `makeFoo`, so you could call it `makeRSAMethod` or so if you at set on wanting to change the names.